packages/paste-core/components/description-list/src/DescriptionListSet.tsx

```diff
@@ -6,7 +6,7 @@ import * as React from "react";
 
 export const StyledDescriptionListSet = styled.div(
   css({
-    "& > dd:last-of-type": {
+    ":not(:last-of-type)": {
```

Comment:
question: I'm confused by this change. Can you explain what this does please?

Reply:
```
<DescriptionList>
  <DescriptionListSet>
    <DescriptionListTerm>term1</DescriptionListTerm>
    <DescriptionListDetails>details1a</DescriptionListDetails>
    <DescriptionListDetails>details1b</DescriptionListDetails>
  </DescriptionListSet>
  <DescriptionListSet>
    <DescriptionListTerm>term2</DescriptionListTerm>
    <DescriptionListDetails>details2</DescriptionListDetails>
  </DescriptionListSet>
</DescriptionList>
```

Previously, spacing was being set between Description List Sets by adding a larger bottom margin to the Description Details if it was the last one in the set (aka not on details1a). However that was also adding bottom margin to the entire component (coming from details2). This change makes the spacing that we want (between each DescriptionListSet) come from any non-last DescriptionListSet. So effectively there's no top or bottom margin to the DescriptionList but we still have the inner spacing we need.